docs/components.rst

```diff
@@ -123,8 +126,6 @@ the internal name of a component class would have been :file:`LWO{Name}Component
    components/102-commendation-vendor
    components/104-rail-activator
    components/105-roller
-   components/106-unknown
-   components/107-possessor
```

Comment:
Do we have new info on 107? Because I think it appears in the ComponentsRegistry somewhere

Reply:
https://lcdruniverse.org/lu_packets/lu_packets/raknet/client/replica/bbb/struct.BbbProtocol.html
107 is BBB Protocol

Reply:
Good catch. I'll add 107 in since I technically removed it.  Same with 106

Reply:
Added back 107 and no need to add back 106 since it was just renamed :P